.eslintrc.json

```diff
@@ -51,37 +39,12 @@
               },
               // Exception rules
               {
-                "sourceTag": "lib:experience",
+                "sourceTag": "lib:picking",
```

Comment:
As I understand the only issue is in BAPI feature from the `application` package. But I'm thinking if we can move them to the `auth` package.